<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HOME" ignored="0" name="Test" tests="8" failures="1" timestamp="18 Feb 2019 18:39:41 GMT" time="0.241" errors="0">
  <testcase name="test1" time="0.024" classname="FailedTestCases.TestCases"/>
  <testcase name="test2" time="0.008" classname="FailedTestCases.TestCases"/>
  <testcase name="test3" time="0.026" classname="FailedTestCases.TestCases">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.012" classname="FailedTestCases.TestCases">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.007" classname="FailedTestCases.TestCases">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.004" classname="FailedTestCases.TestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at FailedTestCases.TestCases.test3(TestCases.java:19)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test4" time="0.006" classname="FailedTestCases.TestCases"/>
  <testcase name="test6" time="0.001" classname="FailedTestCases.TestCases"/>
</testsuite> <!-- Test -->
